src/main/scala/org/broadinstitute/dsde/firecloud/core/ExportEntitiesByType.scala

```diff
@@ -55,13 +55,13 @@ class ExportEntitiesByTypeActor(requestContext: RequestContext) extends Actor wi
           allSucceeded match {
             case true =>
               val entities = responses.flatMap(unmarshal[List[EntityWithType]].apply)
-              entityType match {
-                case x if x.endsWith("_set") =>
-                  val rootType = x.replace("_set", "")
-                  val zippedFileName = x + ".zip"
-                  val rootData = TSVFormatter.makeEntityTsvString(entities, rootType)
-                  val membershipData = TSVFormatter.makeMembershipTsvString(entities, entityType)
-                  val zipBytes: Array[Byte] = getZipBytes(filename, membershipData, rootType, rootData)
+              ModelSchema.getCollectionMemberType(entityType) match {
```

Comment:
this PR doesn't modify the code at line 45 above - so I commented here: is what line 45 is doing above still needed?  Isn't that from what it used to do before to get both the set and all of the entities of that set type?